src/Feed/SourceType/Colibo/ColiboFeedType.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Feed\SourceType\Colibo;
+
+use App\Entity\Tenant\Feed;
+use App\Entity\Tenant\FeedSource;
+use App\Feed\FeedOutputModels;
+use App\Feed\FeedTypeInterface;
+use App\Feed\OutputModel\ConfigOption;
+use App\Service\FeedService;
+use FeedIo\Feed\Item;
+use FeedIo\Feed\Node\Category;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Uid\Ulid;
+
+/**
+ * Colibo Intranet Feed.
+ *
+ * @see https://intranet.colibo.com/apidocs
+ * @see https://intranet.colibo.com/apidocs/reference/index
+ */
+class ColiboFeedType implements FeedTypeInterface
+{
+    public const int CACHE_TTL = 3600;
+
+    final public const string SUPPORTED_FEED_TYPE = FeedOutputModels::RSS_OUTPUT;
+
+    public function __construct(
+        private readonly FeedService $feedService,
+        private readonly ApiClient $apiClient,
+        private readonly CacheItemPoolInterface $feedsCache,
+    ) {}
+
+    public function getAdminFormOptions(FeedSource $feedSource): array
+    {
+        $feedEntryRecipients = $this->feedService->getFeedSourceConfigUrl($feedSource, 'allowed-recipients');
+
+        return [
```

Comment:
Why is this here?

Reply:
getAdminFormOptions returns the admin options for the given feed type in the form of an array of array elements.